repo/packages/C/calico/2/config.json

```diff
@@ -0,0 +1,245 @@
+{
+  "properties": {
+    "Framework Settings": {
+      "description": "Configuration options for Calico's DC/OS Framework.",
+      "properties": {
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "calico-install-framework"
+        },
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
```

Comment:
ok cool. So if I understand correctly, always use the full string, because the "extra" `zk`'s will still resolve to real zk's?